app/views/dashboard/autoyast.xml.erb

```diff
@@ -13,6 +13,17 @@
         ]]>
         </source>
       </script>
+      <script>
+        <chrooted config:type="boolean">true</chrooted>
+        <filename>configure_timesyncd.sh</filename>
+        <interpreter>shell</interpreter>
+        <source><![CDATA[
+        #!/bin/sh
+        sed -i -e 's|#NTP=.*|NTP=<%= @controller_node %>|g' /etc/systemd/timesyncd.conf
```

Comment:
I will not change not even a single char of what Throsten suggested [here](https://trello.com/c/Orzhm7rZ) :-D